### PR TITLE
Update azuredeploy.json

### PR DESCRIPTION
**Pull request by AcceleryntSecurityDev**
Change(s):
- updated sensitive parameters to secure string

   Reason for Change(s):
   - security improvement

   Version Updated:
   - No

   Testing Completed:
   - No

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Playbooks/AS_Alert_Spiderfoot_Scan/azuredeploy.json [hunk 1]
```diff
@@ -15,8 +15,8 @@
             "type": "string"
         },
         "SpiderfootAPIKey": {
-            "defaultValue": "Enter your Spiderfoot API key here.",
-            "type": "string"
+            "defaultValue": "",
```
Comment (sreedharande) [inline Playbooks/AS_Alert_Spiderfoot_Scan/azuredeploy.json:18]: @AcceleryntSecurityDev - Delete "Default Value" to make it Required Column

**Review by sreedharande — CHANGES_REQUESTED** [review]
@AcceleryntSecurityDev - Delete "defaultValue" to make it Required Column, as API key is required value for the Playbook

**Review by sreedharande — CHANGES_REQUESTED** [review]
@AcceleryntSecurityDev - SpiderfootURL - make it required by removing defaultValue. Add "metadata": {
            "description": ""
        } that doesn't have default value

**Comment by AcceleryntSecurityDev** [comment]
@sreedharande I am unable to re-request review on this PR for some reason, please review when you get a chance

**Review by sreedharande — APPROVED** [review]

**Pull request merged**